src/pke/examples/iterative-ckks-bootstrapping.py

```diff
@@ -0,0 +1,119 @@
+from openfhe import *
+import math
+import random
+
+def main():
+    IterativeBootstrapExample()
+
+def CalculateApproximationError(result,expectedResult):
```

Comment:
If it's not too much work can you convert the naming style to be more pythonic? 

`CalculateApproximationError` -> `calculate_approximation_error` and `expectedResult` -> `expected_result`

This only applies to python code i.e. files that end in `.py`